packages/@react-spectrum/overlays/src/Modal.tsx

```diff
@@ -66,6 +66,9 @@ let ModalWrapper = forwardRef(function (props: ModalWrapperProps, ref: RefObject
   let wrapperClassName = classNames(
     modalStyles,
     'spectrum-Modal-wrapper',
+    {
+      'spectrum-fullscreenTakeover-Modal-wrapper': typeVariant === 'fullscreenTakeover'
```

Comment:
spectrum-Modal-wrapper--fullscreenTakeover I think is the correct classname to use
spectrum is namespace
Modal is the component
wrapper is the sub component
fullscreenTakeover is the state of the component

Reply:
Sounds good, that makes sense